tests/CMakeLists.txt

```diff
@@ -433,6 +433,36 @@ target_link_libraries(arangodbtests
   fmt
 )
 
+set(ARANGODB_SEPP_SOURCES
```

Comment:
I am happy if the sepp executable gets built whenever arangodbtests is built, as it is currently implemented.
I don't think this will cause issues for release packages, because there we don't build the tests. 
The only potential issue I see here is that the extra executable will take up disk space and increase link times, and as it is statically linked, potentially about 1 GB with debug symbols.
I don't expect this to be a problem, but we need to check.

Reply:
Sepp can now only be built if the cmake variable `BUILD_SEPP` is enabled.